packages/devtools_app/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ description: Main package implementing web-based performance tooling for Dart an
 # When publishing new versions of this package be sure to publish a new version
 # of package:devtools as well. package:devtools contains a compiled snapshot of
 # this package.
-version: 0.2.2
+version: 0.2.3
```

Comment:
please revert this change - I'll bump the version in my release prep cl.

 changelog entries are fine to stay